client/src/theme/dark/_smui-theme.scss

```diff
@@ -33,3 +33,7 @@ a:visited {
 .error {
   color: theme.$error;
 }
+
+.catfish {
+  color: #d32f2f;
+}
```

Comment:
this code is duplicated in `ReceiveRole` maybe it would be best to move all three role colors into the theme